.github/workflows/coverage.yml

```diff
@@ -25,6 +25,11 @@ jobs:
         pip install -U coverage
         pip install -U codecov
         pip install -r requirements.txt
+        pip install -r bin/requirements.txt
+        echo "y" | pip uninstall epispot
+    - name: Build from source
+      run: |
+        python setup-nightly.py install
```

Comment:
Use `develop` not `install` with setup.py

Reply:
`install` installs a stable and unchanging version of the package whereas `develop` means that the package can change while being tested. The goal here is to simulate an actual installation from PyPI where nothing in the package will change and the entire source code remains the same. Therefore, we use `install` in both `coverage.yml` and `build.yml`.